experimental_official_dockerfiles/master/cleanup_cuda_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# ??? Magic ???
+
+# Delete uneccessary static libraries, which we don't need for some reason.
```

Comment:
I don't understand these; they're long-time members. Do we not use the *.a files? Why would we need to keep libcudart and nothing else? Why can we remove libcudnn_static_v8.a in the other directory?

@perfinion